meta/markenzoo/contao-expo-push-notification-bundle/logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 400 400" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
+    <g transform="matrix(0.941403,0,0,0.932988,23.4388,26.8048)">
+        <path d="M400,-28.73L-24.898,-28.73L-24.898,400L400,-28.73Z" style="fill:rgb(106,174,0);"/>
+    </g>
+    <g transform="matrix(-0.941403,1.15289e-16,-1.14258e-16,-0.932988,376.561,373.195)">
+        <path d="M400,-28.73L-24.898,-28.73L-24.898,400L400,-28.73Z" style="fill:rgb(89,150,0);"/>
+    </g>
+    <path d="M326.664,110.907L400,184.049L400,400L203.346,400.175L82.326,284.693L82.327,284.683C85.526,287.333 89.631,288.926 94.105,288.926L182.235,288.926L208.12,320.128L234.313,288.926L312.588,288.926C322.789,288.926 331.07,280.644 331.07,270.443L331.07,122.879C331.07,118.314 329.412,114.133 326.664,110.907ZM203.231,169.279C215.813,151.516 236.529,153.16 247.628,162.326C258.728,171.492 260.653,191.137 247.628,208.155C237.653,221.189 219.88,235.653 203.231,244.819C186.583,235.653 168.627,221.904 158.835,208.155C146.403,190.7 147.735,171.492 158.835,162.326C169.934,153.16 189.274,150.27 203.231,169.279Z" style="fill:rgb(76,118,19);"/>
+    <path d="M331.07,122.879C331.07,112.678 322.789,104.396 312.588,104.396L94.105,104.396C83.904,104.396 75.622,112.678 75.622,122.879L75.622,270.443C75.622,280.644 83.904,288.926 94.105,288.926L182.235,288.926L208.12,320.128L234.313,288.926L312.588,288.926C322.789,288.926 331.07,280.644 331.07,270.443L331.07,122.879ZM203.231,169.279C215.813,151.516 236.529,153.16 247.628,162.326C258.728,171.492 260.653,191.137 247.628,208.155C237.653,221.189 219.88,235.653 203.231,244.819C186.583,235.653 168.627,221.904 158.835,208.155C146.403,190.7 147.735,171.492 158.835,162.326C169.934,153.16 189.274,150.27 203.231,169.279Z" style="fill:white;"/>
+</svg>
```

Comment:
Please minify the SVG as explained in the README.

Reply:
Done in ae124f3